osu.Game/Overlays/BeatmapSet/BeatmapPicker.cs

```diff
@@ -88,6 +89,12 @@ public BeatmapPicker()
                                     Origin = Anchor.BottomLeft,
                                     Font = OsuFont.GetFont(size: 17, weight: FontWeight.Bold)
                                 },
+                                guestMapperContainer = new FillFlowContainer
```

Comment:
This can probably be a `LinkFlowContainer` directly. `FillFlowContainer` with one child seems pointless.